packages/utils/src/assets-utils/README.md

```diff
@@ -0,0 +1,102 @@
+# Fuel asset list
+
+This package contains the list of assets and metadata, for the Fuel Network.
+
+This package is made by projects that want to consume metadata from assets like, `name`, `symbol`, `icon` and
+`brigde` informations for those assets that support it.
+
+### Install
+
+`npm install @fuel-ts/utils/assets-utils`
+
+### Usage
+
+```ts
+import assets from "@fuel-ts/utils/assets-utils";
```

Comment:
We might not need an entry point, even _if_ we don't move `assets` inside `account`.

```ts
import { assets } from "@fuel-ts/utils";
```

Reply:
Cool, will remove it